tests/mock_web_api_server.py

```diff
@@ -253,9 +257,22 @@ def stop(self):
         self.join()
 
 
+class ReceivedRequestsHandler:
```

Comment:
this is a matter of preference but this class does not handle anything but it just stores the received requests. If I named this, it would be a simpler one such as "ReceivedRequests" or "SavedReceivedRequests"

Reply:
Agree 💯